docs/source/conceptual/tensor_parallelism.md

```diff
@@ -0,0 +1,13 @@
+# Tensor Parallelism
+
+Tensor Paralellism (also called horizontal model paralellism) is a technique used to fit a large model in multiple GPUs. Model parallelism enables large model training and inference by putting different layers in different GPUs (also called ranks). Intermediate outputs between ranks are sent and received from one rank to another in a synchronous or asynchronous manner. When multiplying input with weights for inference, multiplying input with weights directly is equivalent to dividing weight matrix column-wise, multiplying each column with input separately, and then concatenating the separate outputs like below 👇 
```

Comment:
* Typo in a couple of places: Parallelism -> Parallelism
* We discuss tensor parallelism (horizontal), but the explanation corresponds to pipeline parallelism (vertical). As per my understanding, tensor parallelism sends parts of a tensor to different GPUs (not layers). I would just explain tensor parallelism in this guide to focus on tensor parallelism (https://colossalai.org/docs/concepts/paradigms_of_parallelism/#tensor-parallel helped me here)

Reply:
I explained model parallelism first and since tensor parallelism is sub-topic of this I thought I'd mention that.

Reply:
I removed MP.